plugins/epos/plugin.py

```diff
@@ -91,6 +91,19 @@ def __init__(self, item_id, oai_base=None, lang="en", config=None):
         self.terms_access_protocols = ast.literal_eval(
             self.config[self.name]["terms_access_protocols"]
         )
+        self.metadata_standard = ast.literal_eval(
+            self.config[self.name]["metadata_standard"]
+        )
+        self.fairsharing_username = ast.literal_eval(
+            self.config["fairsharing"]["fairsharing_username"]
+        )
+
+        self.fairsharing_password = ast.literal_eval(
+            self.config["fairsharing"]["fairsharing_password"]
+        )
+        self.fairsharing_path = ast.literal_eval(
+            self.config["fairsharing"]["fairsharing_path"]
```

Comment:
there should be a fallback when getting to get parameters that are optional, e.g. username and password. Have you tried if these statements fail when username and password are not defined?

Reply:
If they are defined as an empty list the tool uses the premade list file, but the program fails when the path does not take to any file, probably should give you 0 points but change the msg

Reply:
Fixed in commit Improved test r1_3_01m